src/GiraffeGenerator.Core/CodeGen.fs

```diff
@@ -46,7 +20,11 @@ let private createRouteHandler path method =
         method.Method.ToUpperInvariant() |> identExpr
 
     let serviceCall = service method.Name
-    if method.Parameters.IsSome then verb >=> routeBind path serviceCall else verb >=> route path >=> serviceCall
+    let hasPathParameters =
+        method.Parameters
+        |> Option.map (Map.toSeq >> Seq.map fst >> Seq.map ((=) Path) >> Seq.reduce (||))
```

Comment:
Being a functional is not a goal here =)
Seems like there was some more complex condition which was simplified during the development, but I didn't notice that it may be simplified further to `Option.bind (Map.containsKey Path)`. Thanks!

Reply:
Done